doc/submitter_guide.md

```diff
@@ -312,6 +312,41 @@ Date/Time Format][datetime_format].
 
 Example: `"2020-08-14T23:41:54+00:00"`
 
+#### Issues
+
+##### `build_valid`
+The status to assign to incident builds
+
+Example: `false`
+
+##### `test_status`
+The Status to assign to incident tests, it can be one of the following
+
+* `ERROR` - the test is faulty, the status of the tested code is unknown.
+* `FAIL` - the test has failed, the tested code is faulty.
+* `PASS` - the test has passed, the tested code is correct.
+* `DONE` - the test has finished successfully, the status of the tested code
+  is unknown.
+
+##### `culprit`
+The layers of the execution stack responsible for the issue
+It can be one of the following properties
+
+* `Code` - the built or tested code, value is boolean
+* `Tool` - the static analyzer, the build toolchain, the test, etc
+* `Harness` - the system controlling the execution of the build.
```

Comment:
Each of these properties is all-lower-case, and should be listed as such, as anything else won't be accepted by the schema. All these properties are boolean, so perhaps just mention that in the paragraph above.